src/dogecoin-fees.cpp

```diff
@@ -18,6 +18,50 @@
 #endif
 
 #ifdef ENABLE_WALLET
+
+CFeeRate GetDogecoinFeeRate(int priority)
+{
+    switch(priority)
+    {
+    case SUCH_EXPENSIVE:
+        return CFeeRate(COIN / 100 * 521); // 5.21 DOGE, but very carefully avoiding floating point maths
+    case MANY_GENEROUS:
+        return CFeeRate(CWallet::minTxFee.GetFeePerK() * 100);
+    case AMAZE:
+        return CFeeRate(CWallet::minTxFee.GetFeePerK() * 10);
+    case WOW:
+        return CFeeRate(CWallet::minTxFee.GetFeePerK() * 5);
+    case DOUBLE:
+        return CFeeRate(CWallet::minTxFee.GetFeePerK() * 2);
+    case NORMAL:
+    default:
+        break;
+    }
+    return CWallet::minTxFee;
+}
+
+const char *GetDogecoinPriorityLabel(int priority)
+{
+    switch(priority)
+    {
+    case SUCH_EXPENSIVE:
+        return "Such expensive";
+    case MANY_GENEROUS:
+        return "Many generous";
+    case AMAZE:
+        return "Amaze";
+    case WOW:
+        return "Wow";
+    case DOUBLE:
+        return "Double";
+    case NORMAL:
+        return "Normal";
```

Comment:
Technically, this is the "minimum". Maybe we should call it that.

Reply:
Done